src/test/java/seedu/address/logic/parser/property/AddPropertyCommandParserTest.java

```diff
@@ -0,0 +1,274 @@
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.INVALID_PROPERTY_NAME;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.INVALID_PROPERTY_PROPERTY_TYPE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.INVALID_PROPERTY_SELLER_ID;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_ADDRESS_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_ADDRESS_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_ASKING_PRICE_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_ASKING_PRICE_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_IS_RENTAL_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_IS_RENTAL_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_NAME_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_NAME_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_SELLER_ID_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_SELLER_ID_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_TYPE_DESC_ANCHORVALE;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.PROPERTY_TYPE_DESC_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_ADDRESS_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_ASKING_PRICE_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_IS_RENTAL_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_NAME_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_PROPERTY_TYPE_BEDOK;
+import static seedu.address.logic.commands.property.PropertyCommandTestUtil.VALID_PROPERTY_SELLER_ID_BEDOK;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.model.property.Property.DEFAULT_PROPERTY_ID;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.property.AddPropertyCommand;
+import seedu.address.model.id.Id;
+import seedu.address.model.price.Price;
+import seedu.address.model.property.IsRental;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.PropertyName;
+import seedu.address.model.property.PropertyType;
+import seedu.address.testutil.property.PropertyBuilder;
+
+public class AddPropertyCommandParserTest {
```

Comment:
The methods in this class seem very lengthy and perhaps you can consider abstracting or reducing the length to make it clearer for future readers.